test/ode/ode_dense_tests.jl

```diff
@@ -211,6 +211,9 @@ regression_test(CFRLDDRK64(), 3.0e-5, 3.0e-5)
 # TSLDDRK74
 regression_test(TSLDDRK74(), 3.0e-5, 3.0e-5)
 
+# CKLLDDRK43_2
+regression_test(CKLLDDRK43_2(), 7.0e-5,3.5e-5)
```

Comment:
@ranocha Here I have chosen the minimum tolerances so that the method passes the tests. Could you suggest some more accurate values?

Reply:
No, that approach is just fine. The tests in this file are some regression tests, so you should choose the tolerances exactly in the way you did - implement the algorithm, test it carefully, and choose the tolerances such that they are only a little bit bigger than absolutely necessary to let the tests pass.

Reply:
Yup, these regression tests are just checking for if the interpolation goes wonky some time in the future, for example if someone accidentally aliases `k` when `calck=true`.